### PR TITLE
Fix rabbitmq for non-root container

### PR DESCRIPTION
**Pull request by javsalgar**
**What this PR does / why we need it**:
This PR makes the chart compatible with Rabbitmq non-root container

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/rabbitmq/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: rabbitmq
-version: 1.1.9
+version: 2.0.0
 appVersion: 3.7.6-ol-7
```
Comment (tompizmor) [inline stable/rabbitmq/Chart.yaml]: can you change the app version to only `3.7.6`?

File: stable/rabbitmq/templates/statefulset.yaml [hunk 2]
```diff
@@ -56,18 +56,16 @@ spec:
          - |
             #persist the erlang cookie in both places for server and cli tools
             echo $RABBITMQ_ERL_COOKIE > /opt/bitnami/rabbitmq/var/lib/rabbitmq/.erlang.cookie
-            cp /opt/bitnami/rabbitmq/var/lib/rabbitmq/.erlang.cookie /root
-
+            cp /opt/bitnami/rabbitmq/var/lib/rabbitmq/.erlang.cookie /opt/bitnami/rabbitmq/.rabbitmq/
             #change permission so only the user has access to the cookie file
-            chmod 400 /root/.erlang.cookie
+            chmod 400 /opt/bitnami/rabbitmq/.rabbitmq/.erlang.cookie
```
Comment (tompizmor) [inline stable/rabbitmq/templates/statefulset.yaml:61]: it it not necessary to specify that the erlang cookie is in a new path?

Comment (javsalgar) [reply stable/rabbitmq/templates/statefulset.yaml:61]: No, because $HOME is set as /opt/bitnami/rabbitmq/.rabbitmq/

**Review by tompizmor — APPROVED** [review]
/lgtm

**Comment by dpeterka** [comment]
@javsalgar this appears may have broken the chart:
`cp: cannot create regular file '/opt/bitnami/rabbitmq/.rabbitmq/': Not a directory`

File: stable/rabbitmq/templates/statefulset.yaml [hunk 3]
```diff
@@ -56,18 +56,16 @@ spec:
          - |
             #persist the erlang cookie in both places for server and cli tools
             echo $RABBITMQ_ERL_COOKIE > /opt/bitnami/rabbitmq/var/lib/rabbitmq/.erlang.cookie
-            cp /opt/bitnami/rabbitmq/var/lib/rabbitmq/.erlang.cookie /root
-
+            cp /opt/bitnami/rabbitmq/var/lib/rabbitmq/.erlang.cookie /opt/bitnami/rabbitmq/.rabbitmq/
```
Comment (dpeterka) [inline stable/rabbitmq/templates/statefulset.yaml:59]: Please update with `mkdir -p /opt/bitnami/rabbitmq/.rabbitmq`

**Pull request merged**